agents/evacuate/fence_evacuate.py

```diff
@@ -397,6 +396,11 @@ def main():
 		del options["--domain"]
 
 
+	# Disable insecure-certificate-warning message
+	if "--insecure" in options:
+		import urllib3
+		urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
```

Comment:
Why would we disable the insecure requests warnings here? If you explicitely add --insecure I think it is fine to get these messages

Reply:
To avoid spamming the logs.

That's how we do it for the other agents, and it makes sense as you have to use the parameter to make it accept insecure certs.

Reply:
I think that is silly tbh. By doing this, folks might use --insecure and never realize the implications of it.